### ISSUE TITLE
Missing service file after update

### ISSUE THREAD

**Issue by sonyarianto** [arangodb/arangodb#9128]
## My Environment

* __ArangoDB Version__:        3.4.5 to 3.4.6
* __Storage Engine__:             RocksDB
* __Deployment Mode__:       Single Server
* __Deployment Strategy__:   ArangoDB Starter
* __Configuration__:               normal
* __Infrastructure__:               own
* __Operating System__:        CentOS 7
* __Total RAM in your machine__:        24Gb
* __Disks in use__:        HDD
* __Used Package__:              SUSE or RedHat .rpm

## Component, Query & Data

__Affected feature__:
yum update

__AQL query (if applicable)__:

__AQL explain (if applicable)__:

__Dataset__:

__Size of your Dataset on disk__:

__Replication Factor & Number of Shards (Cluster only)__:

## Steps to reproduce

1. I am running version 3.4.5 on CentOS 7 with 3TB HDD with 24Gb RAM. I do yum update and seems on repo there is 3.4.6
2. Update success but after that the file /etc/systemd/system/arangodb3.service is gone and ArangoDB also dead.
3. Lucky I have backup of that file and then I put the file again and I restart the server with (shutdown -r now)

__Problem__:
Why the /etc/systemd/system/arangodb3.service is gone after update (this always happen to me)
and now I can't access log page from the ArangoDB Web interface

__Expected result__:
If update should be smooth mannn :)

Additional Data:
Latest log
```
[root@thanos]# tail -F /var/log/arangodb3/arangod.log
2019-05-29T01:46:45Z [5482] INFO ArangoDB has been shut down
2019-05-29T02:40:18Z [5475] INFO ArangoDB 3.4.6 [linux] 64bit, using jemalloc, build tags/v3.4.6-0-g4f6d3de399, VPack 0.1.33, RocksDB 5.16.0, ICU 58.1, V8 5.7.492.77, OpenSSL 1.1.0j  20 Nov 2018
2019-05-29T02:40:18Z [5475] INFO detected operating system: Linux version 3.10.0-957.12.2.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-36) (GCC) ) #1 SMP Tue May 14 21:24:32 UTC 2019
2019-05-29T02:40:18Z [5475] INFO {authentication} Jwt secret not specified, generating...
2019-05-29T02:40:18Z [5475] INFO using storage engine rocksdb
2019-05-29T02:40:18Z [5475] INFO {cluster} Starting up with role SINGLE
2019-05-29T02:40:18Z [5475] INFO {syscall} file-descriptors (nofiles) hard limit is 131072, soft limit is 131072
2019-05-29T02:40:18Z [5475] INFO {authentication} Authentication is turned on (system only), authentication for unix sockets is turned on
2019-05-29T02:40:24Z [5475] INFO using endpoint 'http+tcp://0.0.0.0:8529' for non-encrypted requests
2019-05-29T02:40:25Z [5475] INFO ArangoDB (version 3.4.6 [linux]) is ready for business. Have fun!
```
service file
```
[Unit]
Description=ArangoDB database server
After=sysinit.target sockets.target timers.target paths.target slices.target network.target syslog.target

[Service]
# we could use another type for more reliable reporting
Type=simple

PermissionsStartOnly=true
User=arangodb
Group=arangodb

# system limits
LimitNOFILE=131072
LimitNPROC=131072
TasksMax=131072

PIDFile=/var/run/arangodb3/arangod.pid
Environment=GLIBCXX_FORCE_NEW=1

# Protect users from making their installation unusable by
# starting arangod with wrong permissions (e.g. as root).
# This will reset the permissions to the working default.
ExecStartPre=/usr/bin/install -g arangodb -o arangodb -d /var/tmp/arangodb3
ExecStartPre=/usr/bin/install -g arangodb -o arangodb -d /var/run/arangodb3
ExecStartPre=/bin/chown -R arangodb:arangodb /var/log/arangodb3
ExecStartPre=/bin/chmod 700 /var/log/arangodb3
ExecStartPre=/bin/chown -R arangodb:arangodb /var/lib/arangodb3
ExecStartPre=/bin/chmod 700 /var/lib/arangodb3
ExecStartPre=/bin/chown -R arangodb:arangodb /var/lib/arangodb3-apps
ExecStartPre=/bin/chmod 700 /var/lib/arangodb3-apps

ExecStart=/usr/bin/numactl --interleave=all /usr/sbin/arangod --pid-file /var/run/arangodb3/arangod.pid --temp.path /var/tmp/arangodb3 --log.foreground-tty true

TimeoutStopSec=3600
TimeoutSec=3600

Restart=on-failure
RestartSec=5

[Install]
WantedBy=multi-user.target
```

**Comment by maxkernbach** [comment]
Hi @sonyarianto,

thanks for reporting, these are two independent bugs.

1. The CentOS upgrade issue is already listed in the [Known Issues in ArangoDB 3.4
](https://www.arangodb.com/docs/stable/release-notes-known-issues34.html#upgrading) documentation.

2. The "LOGS" page in the UI not being available is another bug, which is caused by the 3.4.6 package itself. It is not related to the package upgrade. As you already noticed, a workaround is to access the log file located at path `/var/log/arangodb3/arangod.log`.

We will let you know when these bugs are fixed.

**Comment by sonyarianto** [comment]
@maxkernbach thank you for the answer, I appreciate the quick response.

OK for the missing service file, I will do my workaround like above for a while, and for the Web UI logs page bug, I will check it via server console.

I hope these will be fixed soon :) Thanks a lot and have a nice day.

**Comment by maxkernbach** [comment]
Bug **2** (broken logs UI) will be fixed in the next patch release 3.4.7 (see https://github.com/arangodb/arangodb/issues/9163).

**Comment by sonyarianto** [comment]
@maxkernbach thank you

**Comment by maxkernbach** [comment]
@sonyarianto

FYI: The fix for Bug 2 (broken logs UI) is now available in 3.4.7.
The latest version can be found in the [download section](https://www.arangodb.com/download-major/).

The CentOS upgrade issue has not been addressed yet.

**Comment by sonyarianto** [comment]
@maxkernbach Hello, thank you, yes I notice that Web UI (log page bug) now fixed on 3.4.7. And yes, during upgrade my service file missing :( But anyway I love using ArangoDB :)

**Comment by maxkernbach** [comment]
Closing this issue since the fix for upgrading the rpm package under CentOS is available in versions 3.4.8, 3.5.0.

Upgrading works from these versions on again i.e.:
- 3.4.8 -> 3.4.9
- 3.4.8 -> 3.5.0 -> 3.5.1
